layouts/page/superpowers.html

```diff
@@ -0,0 +1,276 @@
+                    </div>
+                    <div class="pl-6 pr-12">
+                        <h3 class="font-normal text-gray-800 text-3xl mt-0">
+                            <a href="{{ relref . "/docs/guides/continuous-delivery" }}">
+                                <span class="align-middle text-blue-700 transition-all hover:mr-1">Delivery</span>
+                                <span class="text-blue-400 text-lg">&rarr;</span>
+                            </a>
+                        </h3>
+                        <p class="text-lg text-gray-700 mb-12 pb-12 border-b border-gray-1 leading-relaxed">
+                            Continuously ship application and infrastructure updates using common
+                            workflows. Integrates with your CI/CD toolchain to improve velocity and
+                            visibility into infrastructure dependencies and state.
+                        </p>
+                    </div>
+                </li>
+                <li class="md:w-1/2 flex items-start">
+                    <div class="bg-blue-100 border-4 border-blue-200 rounded-full p-6" style="fill: #4387c7;">
+                        <svg class="w-12 h-12" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 640 512"><defs><style>.fa-secondary{opacity:.4}</style></defs><path d="M104 320H56v-57.59A38.45 38.45 0 0 1 94.41 224H296v-64h48v64h201.59A38.46 38.46 0 0 1 584 262.41V320h-48v-48H344v48h-48v-48H104z" class="fa-secondary"/><path d="M128 352H32a32 32 0 0 0-32 32v96a32 32 0 0 0 32 32h96a32 32 0 0 0 32-32v-96a32 32 0 0 0-32-32zM384 0H256a32 32 0 0 0-32 32v96a32 32 0 0 0 32 32h128a32 32 0 0 0 32-32V32a32 32 0 0 0-32-32zm224 352h-96a32 32 0 0 0-32 32v96a32 32 0 0 0 32 32h96a32 32 0 0 0 32-32v-96a32 32 0 0 0-32-32zm-240 0h-96a32 32 0 0 0-32 32v96a32 32 0 0 0 32 32h96a32 32 0 0 0 32-32v-96a32 32 0 0 0-32-32z" class="fa-primary"/></svg>
+                    </div>
+                    <div class="pl-6 pr-12">
+                        <h3 class="font-normal text-gray-800 text-3xl mt-0">
+                            <a href="{{ relref . "/docs/intro/concepts" }}">
+                                <span class="align-middle text-blue-700 transition-all hover:mr-1">Architecture</span>
+                                <span class="text-blue-400 text-lg">&rarr;</span>
+                            </a>
+                        </h3>
+                        <p class="text-lg text-gray-700 mb-12 pb-12 border-b border-gray-1 leading-relaxed">
+                            Stop reinventing the wheel with each new microservice component and create
```

Comment:
```suggestion
                            Codify application and infrastructure patterns using components, and share them using package managers. Leverage an ecosystem of libraries with built-in best practices.
```

Reply:
looks fine - will need to see in preview if the extra text causes issues.